vue/src/TagmanagerTrackingCode/TagManagerTrackingCode.less

```diff
@@ -0,0 +1,11 @@
+div[anchor='tagmanager'] div[matomo-tagmanager-tracking-code] {
+  .list-style-decimal {
```

Comment:
We should either create `.list-style-decimal-inside` in core and have it available to plugins, not overwrite it per plugin, or the position inside the default in core and still avoid overriding it in plugins.

Reply:
I wouldn't be opposed to adding that, but we're trying to do a quick turnaround on this improvement.

Reply:
Fair enough, though that shouldn't justify keeping or increasing technical debt by tiny customisations in each plugin over a product-level consistent approach.